### PR TITLE
add simple scheme define (aliases)

### PR DESCRIPTION
**Pull request by nittka**
simple scheme defines referrable as assignments, allowing two more integration test cases to pass

I am not sure what name and value may look like, so the SimpleSchemeAssignment rule may still be refined

### CODE REVIEW

File: org.elysium.test/src/org/elysium/test/regression/GrammarRegressions.xtend [hunk 1]
```diff
@@ -25,6 +25,25 @@ class GrammarRegressions {
...
+		model.assertNoErrors
+
+		val define=model.expressions.head
+		Assert.assertTrue(define instanceof Assignment)
+
+		val referencingAssignment=model.expressions.last
+		Assert.assertTrue(referencingAssignment instanceof Assignment)
+		val ref=(referencingAssignment as Assignment).value
+		Assert.assertTrue(ref.toString,ref instanceof Reference)
+		Assert.assertEquals("anAlias", (ref as Reference).assignment.name)
```
Comment (thSoft) [inline org.elysium.test/src/org/elysium/test/regression/GrammarRegressions.xtend]: Could you extract the alias name? I know abstraction should be limited in tests, but simple concatenation of string constants isn't too much indirection (especially with Xtend's template expressions).

File: org.elysium/src/org/elysium/LilyPond.xtext [hunk 2]
```diff
@@ -14,11 +14,12 @@ import "http://www.eclipse.org/emf/2002/Ecore" as ecore
 
 LilyPond: expressions+=Expression*;
 
-Expression: Assignment | CommonExpression;
+Expression: Assignment | SimpleSchemeAssignment | CommonExpression;
 
 CommonExpression: Command | Block | Scheme | Number | Text;
 
 Assignment: name=(SchemeIdentifier | STRING) "=" value=Expression;
+SimpleSchemeAssignment returns Assignment: "#" "(" "define" name=ID value=Text ")";
```
Comment (thSoft) [inline org.elysium/src/org/elysium/LilyPond.xtext]: Nice solution!

**Comment by nittka** [comment]
I used the markup/markuplines pattern for markuplist as well, which seems to fix further test cases.
I also analysed errors I got from the snippet repository files. For two of the error classes I added grammar regression tests which do not pass yet.

**Comment by thSoft** [comment]
100% test success comes closer as closer...

**Pull request merged**